nuitka/Reports.py

```diff
@@ -85,6 +86,20 @@ def writeCompilationReport(report_filename):
 
         root.append(module_xml_node)
 
+    modules_node = TreeXML.Element("missing-modules")
+
+    for module_name in OrderedSet(
```

Comment:
That iteration over an `OrderedSet`  does not really contribute, or does it, could just as well iterate over the list generator, which probably could also be a generator.

Reply:
Converting a list to an Orderset that is then iterated over has no effect, really.